doc/sphinx/practical-tools/utilities.rst

```diff
@@ -92,14 +92,39 @@ CoqMakefile
   is a makefile for ``GNU Make`` with targets to build the project
   (e.g. generate .vo or .html files from .v or compile .ml* files)
   and install it in the ``user-contrib`` directory where the |Coq|
-  library is installed. Run ``make`` with the ``-f CoqMakefile``
-  option to use ``CoqMakefile``.
+  library is installed.
 
 CoqMakefile.conf
   contains make variables assignments that reflect
   the contents of the ``_CoqProject`` file as well as the path relevant to
   |Coq|.
 
+The recommended approach is to wrap ``CoqMakefile`` into a standard ``Makefile`` of the form:
+
+::
+
+    all: CoqMakefile
+            +make -f CoqMakefile all
+
+    clean: CoqMakefile
+            +make -f CoqMakefile clean
+            rm -f Makefile.coq
+
+    CoqMakefile: _CoqProject
+            $(COQBIN)coq_makefile -f _CoqProject -o CoqMakefile
+
+    _CoqProject: ;
+
+    %: CoqMakefile
+            +make -f CoqMakefile $@
```

Comment:
Should we show a mechanism like https://github.com/coq/coq/blob/e583be62b74d71b5af159700e3a31f78fec9a7d2/Makefile.make#L166-L178 so that `make a b` results in a single subcall to make? Also
```suggestion
            $(MAKE) -f CoqMakefile $@
```
is the recommended way.